README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
 
-### Features
+## Features
 
-* Classic F-Droid style, without cards or inappropriate animations
-* Fast repositories synchronization under good network conditions
-* Built upon standard Android components and a minimal set of dependencies
-* Made with attention to details, both in program logic and design
+* Classic F-Droid style.
+* No cards or inappropriate animations.
+* All apps shown in "Available".
```

Comment:
I think "classic F-Droid style" implies that. This point is not necessary.

Reply:
Which one? People might not know what "Classic F-Droid style" is.

Reply:
They I don't think it should be described this way.

"A single-listed catalog of applications" or something like that is better to describe that, instead of pointing out the "Available" tab.

Reply:
"Uninterrupted listing of apps" maybe. G-Droid made a big point of this. Removed.